be/src/formats/orc/apache-orc/c++/src/ColumnReader.cc

```diff
@@ -1245,6 +1295,13 @@ void ListColumnReader::seekToRowGroup(PositionProviderMap* positions) {
     }
 }
 
+void ListColumnReader::lazyLoadSeekToRowGroup(PositionProviderMap* positions) {
+    // If List support lazy load, here need to check
```

Comment:
here to check what?

Reply:
typo, just means If we support list/map/struct's lazy load, we need to handle `null` carefully.